src/core/portal/portal-directives.ts

```diff
@@ -0,0 +1,110 @@
+import {Portal} from './portal';
```

Comment:
Why not put this in `src/directives/portal`? Is this not something we want to expose publicly?